doc/news/changes.h

```diff
@@ -233,6 +233,14 @@ inconvenience this causes.
 <h3>Specific improvements</h3>
 
 <ol>
+ <li> Fixed: Overlapping block Jacobi. This fixes a bug in the RelaxationBlock
```

Comment:
Could you please remove the "Overlapping block Jacobi" part? We should have the news message as short as possible and I think that the remainder explains it perfectly (it is also not the usual style in other messages).